### DISCUSSION TITLE
Upgrade azuracast through Linode

### DISCUSSION THREAD

**Discussion by BalderdashBoys** [AzuraCast/AzuraCast#6293] [General]
I run azuacast through Linode. How do I upgrade azuracast to the newest version through linode? Do I use something else and if so how? I am not very tech but trying to learn so any help would be greatly appreciated. Thank you.

**Comment by BusterNeece** [comment]
Check out our update instructions in our documentation: https://docs.azuracast.com/en/getting-started/updates
